index.html

```diff
@@ -6,7 +6,115 @@
   </head>
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
+    <header>
+      <nav>
+        <a href="index.html">
+         <img src="#"alt="Логотип магазина 'Техномарт'">
+        </a>
+      <ul>
+        <li>Посик</li>
```

Comment:
```suggestion
        <li>Поиск</li>
```
Орфографическая ошибка